paperqa/readers.py

```diff
@@ -308,10 +309,9 @@ def read_doc(
     overlap: int = 100,
     page_size_limit: int | None = None,
 ) -> list[Text] | ParsedText | tuple[list[Text], ParsedMetadata]:
-    """Parse a document and split into chunks.
+    """Parse a document and split into chunks without blocking the event loop.
```

Comment:
Let's revert this "without blocking the event loop"

This is really an implementation detail that is innate to all `async` methods, we don't need to document it in our docstring